app/models/bookmark.rb

```diff
@@ -1,3 +1,16 @@
 class Bookmark < ActiveRecord::Base
+
   belongs_to :topic
+  before_validation :httpset
+  validates :url, format: { with: /\Ahttp:\/\/.*(com|org|net|gov)/i,
```

Comment:
This will break if a link is using `https` (https://www.google.com as an example). Can you think of anyway to accept both?